### ISSUE TITLE
Archetype needs to be updated

### ISSUE THREAD

**Issue by keanemind** [mtn/cocoa-eh-hugo-theme#43]
Hugo v0.24 now needs the default archetype to supply title and date for new posts. Hugo doesn't do it automatically anymore. Something like this needs to be added:

```
+++
title = "{{ replace .TranslationBaseName "-" " " | title }}"
date = {{ .Date }}
draft = true
+++
```

**Comment by mtn** [comment]
@keanemind Taking a look!

**Comment by mtn** [comment]
Thanks for reporting this. 💯 

Addressed, pending #44
